### PR TITLE
Added Flake8 to .travis.yml

### PR DESCRIPTION
**Pull request by kamsuri**
I have added Flake8 to .travis.yml, the test fails as flake8 command exits with status '1' due to it's warnings.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ before_script:
   - sleep 3
 script:
   - cd vms
+  - flake8
```
Comment (tapasweni-pathak) [inline .travis.yml]: Did you test this for your fork?

Comment (kamsuri) [reply .travis.yml]: Yes, but it is also failing. You can check it at: https://travis-ci.org/kamsuri/vms/jobs/290791344.

Comment (tapasweni-pathak) [reply .travis.yml]: instead here say ```flake8 .```

Comment (tapasweni-pathak) [reply .travis.yml]: instead here say flake8 .

**Pull request merged**